plugins/mcreator-core/datalists/blockstateproperties.yaml

```diff
@@ -0,0 +1,732 @@
+    min: 0
+    max: 3
+- AGE_5:
+  readable_name: AGE_5
+  type: Integer
+  other:
+    registry_name: age
+    min: 0
+    max: 5
+- AGE_7:
+  readable_name: AGE_7
+  type: Integer
+  other:
+    registry_name: age
+    min: 0
+    max: 7
+- AGE_15:
+  readable_name: AGE_15
+  type: Integer
+  other:
+    registry_name: age
+    min: 0
+    max: 15
+- AGE_25:
+  readable_name: AGE_25
+  type: Integer
+  other:
+    registry_name: age
+    min: 0
+    max: 25
+- BITES:
+  readable_name: BITES
+  type: Integer
+  other:
+    registry_name: bites
+    min: 0
+    max: 6
+- CANDLES:
+  readable_name: CANDLES
+  type: Integer
+  other:
+    registry_name: candles
+    min: 1
+    max: 4
+- DELAY:
+  readable_name: DELAY
+  type: Integer
+  other:
+    registry_name: delay
+    min: 1
+    max: 4
+- DISTANCE:
+  readable_name: DISTANCE
+  type: Integer
+  other:
+    registry_name: distance
+    min: 1
+    max: 7
+- DUSTED:
+  readable_name: DUSTED
+  type: Integer
+  other:
+    registry_name: dusted
+    min: 0
+    max: 3
+- EGGS:
+  readable_name: EGGS
+  type: Integer
+  other:
+    registry_name: eggs
+    min: 1
+    max: 4
+- HATCH:
+  readable_name: HATCH
+  type: Integer
+  other:
+    registry_name: hatch
+    min: 0
+    max: 2
+- LAYERS:
+  readable_name: LAYERS
+  type: Integer
+  other:
+    registry_name: layers
+    min: 1
+    max: 8
+- LEVEL_CAULDRON:
+  readable_name: LEVEL_CAULDRON
+  type: Integer
+  other:
+    registry_name: level
+    min: 1
+    max: 3
+- LEVEL_COMPOSTER:
+  readable_name: LEVEL_COMPOSTER
+  type: Integer
+  other:
+    registry_name: level
+    min: 0
+    max: 8
+- LEVEL_FLOWING:
+  readable_name: LEVEL_FLOWING
+  type: Integer
+  other:
+    registry_name: level
+    min: 1
+    max: 8
+- LEVEL_HONEY:
+  readable_name: LEVEL_HONEY
+  type: Integer
+  other:
+    registry_name: honey_level
+    min: 0
+    max: 5
+- LEVEL:
+  readable_name: LEVEL
+  type: Integer
+  other:
+    registry_name: level
+    min: 0
+    max: 15
+- MOISTURE:
+  readable_name: MOISTURE
+  type: Integer
+  other:
+    registry_name: moisture
+    min: 0
+    max: 7
+- NOTE:
+  readable_name: NOTE
+  type: Integer
+  other:
+    registry_name: note
+    min: 0
+    max: 24
+- PICKLES:
+  readable_name: PICKLES
+  type: Integer
+  other:
+    registry_name: pickles
+    min: 1
+    max: 4
+- POWER:
+  readable_name: POWER
+  type: Integer
+  other:
+    registry_name: power
+    min: 0
+    max: 15
+- RESPAWN_ANCHOR_CHARGES:
+  readable_name: RESPAWN_ANCHOR_CHARGES
+  type: Integer
+  other:
+    registry_name: charges
+    min: 0
+    max: 4
+- ROTATION_16:
+  readable_name: ROTATION_16
+  type: Integer
+  other:
+    registry_name: rotation
+    min: 0
+    max: 15
+- STABILITY_DISTANCE:
+  readable_name: STABILITY_DISTANCE
+  type: Integer
+  other:
+    registry_name: distance
+    min: 0
+    max: 7
+- STAGE:
+  readable_name: STAGE
+  type: Integer
+  other:
+    registry_name: stage
+    min: 0
+    max: 1
+

```

Comment:
Is there a reason for this space?

Reply:
This separates integer and string/enum properties. There also is an empty line between properties of logic and integer types.